rllib/evaluation/rollout_worker.py

```diff
@@ -1220,6 +1220,9 @@ def add_policy(
             KeyError: If the given `policy_id` already exists in this worker's
                 PolicyMap.
         """
+        merged_config = (
```

Comment:
can you use the ``merge_dicts`` util? ``merge_dicts(self.policy_config, config)``

Reply:
@sven1977 Sorry I oversaw Jun's comment as he got proactive with this review! Thanks guys.